### PR TITLE
azurize snm3c

### PR DESCRIPTION
**Pull request by nikellepetrillo**
### Description

This PR makes snm3c multicloud friendly 

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Review by kayleemathews — APPROVED** [review]
Docs are all updated - LGTM!

File: pipelines/skylab/snM3C/snM3C.wdl [hunk 1]
```diff
@@ -24,18 +26,37 @@ workflow snM3C {
...
 
+        String cloud_provider
+
     }
 
     # version of the pipeline
-    String pipeline_version = "2.0.1"
+    String pipeline_version = "2.0.2"
+
+    String snM3C_docker = "m3c-yap-hisat:1.0.0-2.2.1"
```
Comment (ekiernan) [inline pipelines/skylab/snM3C/snM3C.wdl:36]: We had discussed splitting the large snM3C docker up in the future. Will we just move this top-level input for snM3C_docker into the individual tasks?

**Review by ekiernan — APPROVED** [review]
This LGTM. Just had one question about how to implement the docker_path if we decide to split up the large docker.

File: pipelines/skylab/snM3C/snM3C.wdl (hunk 1 above)
Comment (jessicaway) [reply pipelines/skylab/snM3C/snM3C.wdl:36]: We would include the individual dockers here and then change which docker is specified for each task

**Comment by nikellepetrillo** [comment]
Holding off to merge this until we test it out in Azure.

**Comment by nikellepetrillo** [comment]
closing this as snm3c has changed a lot with recent optimizations. a new branch/pr will be made to azurize snm3c.

**Pull request closed without merging**